content/news/devupdate-20180610.md

```diff
@@ -0,0 +1,59 @@
+Development has slowed down a lot recently, but OpenRA is far from dead. I wanted to write something to bridge the news gap until our next official news post, so here's a brief update on a couple of recent projects. We hope to ship these plus many other changes in a new playtest series starting in a few weeks.
+
+One big change affects how we package and distribute our "official" OpenRA builds on Linux. For many years we have automatically generated a deb package, but then relied on downstream packages for other distributions. This has worked well in most respects, but sometimes delays in updates would strand players on older versions, stopping them from playing online. Another long-standing issue on many distros is the (lack of) support for installing playtests and releases at the same time, as players are able to on Windows and macOS.
+
+Our solution to these problems is to adopt the [AppImage](https://github.com/OpenRA/OpenRA/wiki/Appimages) packaging format, which allows us to distribute a portable version of OpenRA that should work on most modern Linux distributions. AppImages can exist alongside normal distro packages and even other versions of OpenRA, which makes it perfect for trying out playtest versions without overwriting the stable release. We will be retiring our deb packages and OBS repository as part of this change, but fear not because stable OpenRA releases are also available on [Flathub](https://flathub.org/apps/details/net.openra.OpenRA) if you prefer a "proper" installation that integrates more closely with your system.
+
+<div style="text-align:center" markdown="1">
+<img src="/images/news/20180610-download.png" width="600" alt="OpenRA Download page">
+
+A sneak-preview of the new download page featuring Linux AppImages.
+
+</div>
+
+Our modding community has continued to be a major focus, driven in a large part by active discussions with and Pull Requests from the modders themselves. We have recently achieved two major modding milestones which we are looking forward to support from the next playtest:
+<ul>
+<li><strong>Feature parity between the Mod SDK and the main OpenRA mods</strong>: the Mod SDK can now package Linux AppImages alongside the existing macOS .apps and Windows installers. Mods now also integrated properly with the online and in-game server lists (no more "Unknown Mod (id)"), and modders can define their own acknowledgements text to be displayed in the in-game Credits dialog.</li>
```

Comment:
2nd sentence: Either "Mods *are* now also integrated ..." or "Mods now also integrate ..." (no d at end of integrate).